packages/containers/TeamList/TeamList.js

```diff
@@ -198,10 +199,12 @@ class TeamListContainer extends Component {
       ...this.props.user?._permissions,
     };
 
+    const teamsLength = activeTabName === 'favorites' ? favoriteTeams.length : teams.length;
```

Comment:
Please, use the TYPE instead of the name.

Reply:
Done